### PR TITLE
Promo and footbar

### PR DESCRIPTION
**Pull request by MarioDena**
# First Solo project for Microverse course, Based on the webpage found here: 
https://www.theodinproject.com/courses/html5-and-css3/lessons/building-with-backgrounds-and-gradients

## Skills used:
1. Gradients.
2. Image embedding.
3. Video loop as background.

*Added a small twist instead of a static image I used a looping video, everything else is as requested.*

## This project was:

* HTML Verified
* Linter Checked
* Made with BEM naming convention
* Build with Sass
* Buid with Flex

Cheers!

### CODE REVIEW

**Comment by MarioDena** [comment]
Fixed the requierements for aproval.

Replaced video for img
Added <footer> tag
README is in the master branch.

File: index.html [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+      <img src="./img/your_verse_hero.jpg" alt="hero" class="backgroundVideo">
+      <!--<div class="backgroundVideo">
+        <video
+          autoplay
+          muted
+          loop
+          playsinline
+          width="100"
+          src="./video/heroVid.mp4"
+        ></video>--->
```
Comment (uche-inyama) [inline index.html]: Remove these lines of code. They are commented out?

File: index.html [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+    <link rel="stylesheet" href="./css/main.css" />
+    <link
+      href="https://fonts.googleapis.com/css?family=Lato:300&display=swap"
+      rel="stylesheet"
+    />
+    <title>Apple Landing Page</title>
+  </head>
+  <body>
+    <main class="content">
+      <img src="./img/your_verse_hero.jpg" alt="hero" class="backgroundVideo">
```
Comment (uche-inyama) [inline index.html]: Adding background image, should be done or implemented in your css file. :slightly_smiling_face:

File: index.html (hunk 1 above)
Comment (MarioDena) [reply index.html]: Ill use them later to add this to a portfolio, but yeah I can modify later. 

Will do the changes.

Thanks

File: index.html (hunk 2 above)
Comment (MarioDena) [reply index.html]: Changed!

**Comment by MarioDena** [comment]
Done README is updated in this branch.

**Pull request merged**